RxSwift/Observables/Implementations/Materialize.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Materialize.swift
+//  Rx
+//
+//  Created by sergdort on 08/03/2017.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+
+public protocol EventType {
```

Comment:
If I move to `Event.swift` 

```
/// Represents a sequence event.
///
/// Sequence grammar: 
/// **next\* (error | completed)**
```
Remain the same. Happy to add anything else though.

Not sure what would be appropriate description to `var event: Event<T>`